cmd/containerd-release/build.go

```diff
@@ -0,0 +1,43 @@
+package main
+
+import (
+	"crypto/sha1"
+	"encoding/hex"
+	"fmt"
+	"io"
+	"os"
+	"os/exec"
+	"runtime"
+)
+
+func build() error {
+	out, err := exec.Command("make").CombinedOutput()
+	if err != nil {
+		return fmt.Errorf("%s: %s", err, out)
+	}
+	return nil
+}
+
+const tarFormat = "containerd-%s.%s-%s.tar.gz"
+
+func tarRelease(tag string) (string, error) {
+	path := fmt.Sprintf(tarFormat, tag, runtime.GOOS, runtime.GOARCH)
+	out, err := exec.Command("tar", "-zcf", path, "bin/").CombinedOutput()
+	if err != nil {
+		return "", fmt.Errorf("%s: %s", err, out)
+	}
+	return path, nil
+}
+
+func hash(path string) (string, error) {
+	f, err := os.Open(path)
+	if err != nil {
+		return "", err
+	}
+	defer f.Close()
+	s := sha1.New()
+	if _, err := io.Copy(s, f); err != nil {
+		return "", err
+	}
+	return hex.EncodeToString(s.Sum(nil)), nil
```

Comment:
Can we return go-digest object, so that `text/template` printer can print the hash algorithm as well?